README.md

```diff
@@ -108,10 +87,11 @@ cd sdk/python3
 pip install -e .
 ```
 
+### 2.4 Install CCNP Device Plugin
 For the ccnp device plugin, user can find the installation guide under the 'Installation'
```

Comment:
Follow the CCNP device plugin [installation guide ](https://github.com/intel/confidential-cloud-native-primitives/blob/ed06fdf048d34a6e765a921dba23047f84aaa043/device-plugin/ccnp-device-plugin/README.md).